terraform/modernisation-platform-account/iam.tf

```
@@ -372,4 +372,23 @@ resource "aws_iam_policy" "instance-scheduler" {
 resource "aws_iam_role_policy_attachment" "instance-scheduler" {
   policy_arn = aws_iam_policy.instance-scheduler.arn
   role       = aws_iam_role.instance-scheduler.name
-}
\ No newline at end of file
+}
+
+# Create assumable roles with managed policies
+# This is for the github-actions role within the member SSO accounts to be able to assume this githubreadonly
+# role to make the initial OIDC request through the AWS provider
+module "iam_assumable_roles" {
+  source               = "terraform-aws-modules/iam/aws//modules/iam-assumable-roles"
+  version              = "~> 4.11"
+  max_session_duration = 43200
```

```suggestion
  max_session_duration = 3600
```
Since this role will only be used for pipelines shall we reduce the session duration to 1hour max?